### PR TITLE
Distribution adjustments.

### PR DESCRIPTION
**Pull request by gianm**
- Add DistributionMain main class
- Move distribution tarball stuff to "distribution" instead of "server"
- Remove dependency on druid-indexing-service (no longer needed)

### CODE REVIEW

File: distribution/src/main/scala/com/metamx/tranquility/distribution/DistributionMain.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+object DistributionMain
+{
+  def main(args: Array[String]) {
+    args.headOption match {
+      case Some("server") =>
+        ServerMain.main(args.drop(1))
+
+      case _ =>
+        System.err.println(s"Usage: ${getClass.getCanonicalName} <command> [args]")
```
Comment (drcrallen) [inline distribution/src/main/scala/com/metamx/tranquility/distribution/DistributionMain.scala]: exit(1)?

Comment (gianm) [inline distribution/src/main/scala/com/metamx/tranquility/distribution/DistributionMain.scala]: added

**Comment by drcrallen** [comment]
Why is DistributionMain needed separate from ServerMain?

**Comment by gianm** [comment]
@drcrallen #59 will add another main, and sbt's tarball plugin only likes to have one primary main, so this is future proofing.

**Comment by drcrallen** [comment]
:+1:

**Comment by fjy** [comment]
:+1:

**Pull request merged**